api/src/server/websocket.rs

```diff
@@ -201,34 +207,35 @@ impl StreamHandler<ws::Message, ws::ProtocolError> for ProgramWebsocket {
 
                 ctx.text(response_string);
             }
-            ws::Message::Binary(_) => {}
-            ws::Message::Close(_) => {
+            Ok(ws::Message::Close(_)) => {
                 ctx.stop();
             }
-            ws::Message::Nop => {}
+
+            // Don't do anything with these messages
+            Ok(ws::Message::Binary(_))
+            | Ok(ws::Message::Continuation(_))
+            | Ok(ws::Message::Nop)
+            | Err(_) => {}
```

Comment:
Could we unwrap msg before the match so each arm does not need the ok?

Reply:
Actually, I think I prefer having it this way. Otherwise, you need a nested `match`, which IMO is harder to read.